Codeinwp/neve
# Fix Infinite Scroll out of order posts

### Summary


I added a lock to prevent Infinite Scroll from requesting more posts until the previous request is not finished.

⚠️ The JS limit size needs to be increased.

### Will affect visual aspect of the product

NO

### Screenshots 

Normal internet speed

https://github.com/Codeinwp/neve/assets/17597852/57889cc2-b1d9-416c-8263-1048d4033307

Slow 3G

https://github.com/Codeinwp/neve/assets/17597852/8ed8dfb9-fbc3-457c-97d9-48da222d77e1

### Test instructions


1. Make the post and active Infinite Scroll
2. The content will still appear in the right place if a request is behind another.

## Check before Pull Request is ready:



Closes #4146
